app/models/reaction.rb

```diff
@@ -89,15 +89,15 @@ def update_reactable
   end
 
   def bust_reactable_cache
```

Comment:
This is currently happening in an `after_save` callback and I think the majority of these cache busts will occur when a reaction is created which means I believe we need this to be in an `after_commit` callback to ensure that new reactions have been committed to the database before we try to clear their associated caches

Reply:
Do you mean all of these?

`after_save :update_reactable, :bust_reactable_cache, :touch_user, :async_bust`

Reply:
At the very least the `bust_reactable_cache`, we can leave the others for now as to not change everything at once. Just wanted to make sure I was thinking about that right. Some other cache jobs perform `after_save` but they were just busting the model's own cache which doesnt need busting if its new. In this case we are busting other cache's so it needs to be done when its new.

Reply:
@mstruve Should I send these changes?

Reply:
Yep! And make sure to update with master since it looks like there is a conflict :) thank you!

Reply:
Sure @mstruve. Thanks for your review.